src/components/activity/ActivityFormPanel.svelte

```diff
@@ -41,17 +55,24 @@
   let parameterError: string | null = null;
   let startTimeField: FieldStore<string>;
   let endTime: string | null = null;
+  let editingActivityName: boolean = false;
 
   $: if ($selectedActivity) {
+    activityName = $selectedActivity.name;
     argumentsMap = $selectedActivity.arguments;
+    creationTime = $selectedActivity.created_at;
     duration = $selectedActivity.duration;
     id = $selectedActivity.id;
+    sourceSchedulingGoalId = $selectedActivity.source_scheduling_goal_id;
+    lastModifiedTime = $selectedActivity.last_modified_at;
     parent_id = $selectedActivity.parent_id;
     root_activity = getActivityRootParent($activitiesMap, id);
     simulated_activity_id = $selectedActivity.simulated_activity_id;
     startTime = $selectedActivity.start_time;
+    tags = $selectedActivity.tags;
     type = $selectedActivity.type;
     unfinished = $selectedActivity.unfinished;
+    metadata = $selectedActivity.metadata;
   } else {
     argumentsMap = null;
```

Comment:
Should we also set activityName, creationTime, sourceSchedulingGoalId, and lastModifiedTime to `null` here?

Reply:
Believe so, will change that.